solutions/581/01.py

```
@@ -0,0 +1,31 @@
+class Solution:
+    def findUnsortedSubarray(self, nums: List[int]) -> int:
```

Ruff (0.14.7)

Undefined name `List`

(F821)